Adapted from 9467f12 upstream (post v1.30)

diff --git a/gtkdoc/highlight.py b/gtkdoc/highlight.py
index ded674e..6023d33 100644
--- a/gtkdoc/highlight.py
+++ b/gtkdoc/highlight.py
@@ -46,6 +45,5 @@ def highlight_code(code, lang='c'):
 
 
 def append_style_defs(css_file_name):
-    os.chmod(css_file_name, stat.S_IWRITE)
     with open(css_file_name, 'at', newline='\n', encoding='utf-8') as css:
         css.write(HTML_FORMATTER.get_style_defs())
